### ISSUE TITLE
[daq-decode] Enable tests (Trac #1319)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1313]
Enable the test that ensures the example script run.  This needs data in I3_TESTDATA, so when ticket #1312 is cleared, remember to re-enable these tests.

Migrated from https://code.icecube.wisc.edu/ticket/1319
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:13:59", 
    "description": "Enable the test that ensures the example script run.  This needs data in I3_TESTDATA, so when ticket #1318 is cleared, remember to re-enable these tests.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335639558230", 
    "component": "combo core", 
    "summary": "[daq-decode] Enable tests", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-31T05:58:22", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150831T07:12:34: olivas changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150901T05:41:25: 
* olivas commented:

> In changeset 137091:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137091"
> got test-data for this test so i'm enabling it again.  fixes #1313
> ```


* olivas changed resolution from "" to "fixed"
* olivas changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:13:59: 
* olivas commented:

> Milestone offline V15-08-00 deleted


* olivas changed milestone from "offline V15-08-00" to ""
